config/log4j2.properties

```diff
@@ -38,7 +38,10 @@ logger.debezium.level=WARN
 #rootLogger.appenderRef.consoleStderr.ref = consoleStderrAppender
 ############################ log output to console #############################
 ############################ log output to file    #############################
-rootLogger.appenderRef.file.ref = fileAppender
+# hybrid log appender
```

Comment:
Can we keep the old behavior? Let user to choose open `routingAppender` or not?

Reply:
Better to enable by default